var/spack/repos/builtin/packages/basalt/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install basalt
+#
+# You can edit this file again by typing:
+#
+#     spack edit basalt
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Basalt(PythonPackage):
+    """C++11 Graph Storage library"""
```

Comment:
But… it's a python package?

Reply:
Yeeeap. Does this change something apart from the PythonPackage definition?

Reply:
I was thinking that the comment seems to be a bit misleading? I'm basically wondering how to relate a `C+11 Graph Storage library` to Python. E.g. this would make it clearer to me:
```suggestion
    """Python interface of a C++ graph storage library"""
```

Reply:
You have a point there. I think that a slight variation of the title you gave would be a little more representative.
```suggestion
    """C++11 Graph Storage library with Python interface"""
```
@tristan0x What do you think?

Reply:
That's good. Thank you